### PR TITLE
We are on conda-forge!

### LINKED ISSUE #45: Creat a conda-forge recipe
Would increase the accessibility of the tool. In my experience, many more scientists use conda than pip

I have made one much simpler conda-forge recipe before. @jbusecke might be able to help

### PR DESCRIPTION
**Pull request by jbusecke**
- [x] Closes #45 
 - [x] User visible changes (including notable bug fixes) are documented in `whats-new.rst`

### CODE REVIEW

**Comment by jbusecke** [comment]
Do I have to cover installation instructions for conda somewhere else?

File: docs/installation.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Installation
 
 Notes on how to install
 
+##### Conda
+To install the core package run: `conda install -c conda-forge glidertools`.
 
 ##### PyPI
```
Comment (dhruvbalwada) [inline docs/installation.md:9]: Can you clean up the bit about PyPI install. Specificaly do we prefer that people use used Conda? Maybe say that "and to use pip do ...". Also, (I know this is old text), but can you update the sentence about the correct version of pip, if there is a correct version then which one is it?

Comment (jbusecke) [reply docs/installation.md:9]: How is this?

**Comment by dhruvbalwada** [comment]
Looks great, thanks for doing this!!!

**Pull request merged**